samples/bluetooth/broadcast_audio_sink/Kconfig

```
@@ -28,4 +28,11 @@ config SCAN_DELAY
 	  Time to advertise connectable for a Broadcast Assistant to connect
 	  before starting scanning for Broadcast Sources.
 
+config BT_TARGET_BROADCAST_NAME
+	string "Target Broadcast Device Name"
+	default ""
+	help
+	  Name of target broadcast device. If not empty string, sink device
+	   will only listen to the specified broadcast source. 
```

```suggestion
	  Name of target broadcast device. If not empty string, sink device
	  will only listen to the specified broadcast source. 
```

Based on the implementation, might also be worth stating that it's not case sensitive